### PR TITLE
chore: make repo more community-friendly

### LINKED ISSUE #18: Make repo more community-friendly
**Tasks**:

- [x] Make this repository compliant with [community standards](https://github.com/privacy-scaling-explorations/snark-artifacts/community)
- [x] Update overview
- [x] Add good first issues
- [x] Add more instructions on how to add new artifacts and contribute
- [x] Write a README file for each package

### PR DESCRIPTION
**Pull request by cedoor**
Closes #18

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/NEW_ARTIFACTS.yml
Comment (sripwoud) [inline .github/ISSUE_TEMPLATE/NEW_ARTIFACTS.yml:1]: oh wow what is this? ^^
I had no idea you could have files placeholders in a github template!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+3. Add a test for your change. Only refactoring and documentation changes require no new tests.
+
+4. Make sure to check out the [Style Guide](/CONTRIBUTING.md#style-guide) and ensure that your code complies with the rules.
+
+5. Make the test pass.
+
+6. Commit your changes.
+
+7. Push to your fork and submit a pull request on our `main` branch. Please provide us with some explanation of why you made the changes you made. For new features make sure to explain a standard use case to us.
```
Comment (sripwoud) [inline CONTRIBUTING.md:31]:
```suggestion
1. Fork the repo.

<!-- 2. Run the tests. We only take pull requests with passing tests. -->

<!-- 3. Add a test for your change. Only refactoring and documentation changes require no new tests. -->

<!-- 4. Make sure to check out the [Style Guide](/CONTRIBUTING.md#style-guide) and ensure that your code complies with the rules. -->

<!-- 5. Make the test pass. -->

2. Commit your changes.

3. Push to your fork and submit a pull request on our `main` branch. Please provide us with some explanation of why you made the changes you made. For new features make sure to explain a standard use case to us.
```

We don't have any tests (yet?). So don't you think we should not mention tests?

File: OVERVIEW.md
Comment (sripwoud) [inline OVERVIEW.md:1]: I knew about CODE_OF_CONDUCT and CONTRIBUTING community files.
Is OVERVIEW also a standard?

**Review by sripwoud — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (cedoor) [reply CONTRIBUTING.md:31]: Right!

File: OVERVIEW.md
Comment (cedoor) [reply OVERVIEW.md:1]: Nono, I just changed the name of your previous file 😆 Let me know if it's ok for you.

File: .github/ISSUE_TEMPLATE/NEW_ARTIFACTS.yml
Comment (cedoor) [reply .github/ISSUE_TEMPLATE/NEW_ARTIFACTS.yml:1]: Cool ah? First time I use it! Let's see how it works

**Review by sripwoud — APPROVED** [review]
ACK

**Pull request merged**